SharkCage/CageService/CageService.vcxproj

```diff
@@ -178,6 +178,7 @@ xcopy /y "$(OutDir)$(TargetName).pdb" "$(SolutionDir)post_build_output\"
       <SDLCheck>true</SDLCheck>
       <DisableSpecificWarnings>4834</DisableSpecificWarnings>
       <TreatWarningAsError>true</TreatWarningAsError>
+      <AdditionalOptions>/std:c++latest %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Just curious: why is this needed if nothing else in the CageService project changed?

Reply:
Hmm I think this line was added by mistake...tried to get the release build working, but isn't needed anymore.

Reply:
I looked at it again, without the `/std:c++latest %(AdditionalOptions)` the `optional` is not available in the CageService, but used:
`fatal error C1189: #error:  class template optional is only available with C++17.`